docs/docs/faq.md

```diff
@@ -31,3 +31,17 @@ Currently, Kurtosis only supports Dockerhub. If your project or team requires a
 Does Kurtosis pull a container image down each time I run a package?
 --------------------------------------------------------------------
 Kurtosis will always first check the local cache for a given container image for each `kurtosis run` before pulling the image from an external registry (e.g. Dockerhub).
+
+Will Kurtosis be able to run my package remotely from a private Github repository?
+----------------------------------------------------------------------------------
+No, Kurtosis is currently unable to run packages that reside in a private Github repository. Please file a [Github issue on our repository](https://github.com/kurtosis-tech/kurtosis/issues/new?assignees=&labels=feature+request&projects=&template=feature-request.yml) if you believe you need this workflow!
+
+Can I add multiple services in parallel to my enclave via composition?
```

Comment:
@h4ck3rk3y can you sanity check my language/wording here? 🙏🏻

Reply:
This mostly makes sense!

the only thing to note is that if a package looks like "/a/main.star"

```
def run()...

def getConfig()...
```

then one can techincally

```
a = import_module("/a/main.star")

def run():
   a_config = a.getConfig()
   plan.add_services({"a": a_config})
```